generators/client/templates/angular/src/test/javascript/e2e/admin/administration.spec.ts.ejs

```diff
@@ -63,9 +63,9 @@ describe('administration', () => {
   });
   <%_ } _%>
 
+  <!-- This test tends to fail with Protractor, so we've disabled it. Please enable if it passes for you!
```

Comment:
the comment is not correct, as it should be `//` or `/*` instead

Reply:
Doh!